bin/bst-intend.ts

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env node
 import * as program from "commander";
 import {Global} from "../lib/core/global";
-import {BSTAlexa} from "../lib/client/bst-alexa";
+import {BSTVirtualAlexa} from "../lib/client/bst-virtual-alexa";
 
 program.version(Global.version());
 
 program
     .usage("[options] <Intent> [SlotName=SlotValue...]")
     .option("-u, --url <alexa-skill-url>", "The URL of the Alexa skill to send to - defaults to current proxied skill")
+    .option("-m, --model <interaction-model-path>", "Path to the interaction model file - defaults to ./speechAssets/InteractionModel.json")
```

Comment:
For the interaction model file, I think we should use the default location for the ASK CLI - this is what Amazon is moving all there examples to. It is: `./models/en-US.json`

See example here:
https://github.com/alexa/skill-sample-nodejs-fact/tree/en-US/models